components/brave_wallet/browser/json_rpc_response_parser.h

```diff
@@ -80,6 +80,15 @@ absl::optional<std::string> ConvertMultiUint64InObjectArrayToString(
     const std::vector<std::string>& keys,
     const std::string& json);
 
+namespace solana {
+
+base::OnceCallback<absl::optional<std::string>(const std::string& raw_response)>
+ConverterForGetAccountInfo();
+base::OnceCallback<absl::optional<std::string>(const std::string& raw_response)>
+ConverterForGetProrgamAccounts();
```

Comment:
How about putting in solana_response_parser.h instead?

Reply:
https://github.com/brave/brave-core/pull/15617/commits/1e34bd9363c2e1aa4593c00f3409b8c119ccab29

Reply:
@supermassive Move ConverterForGetAccountInfo too.

Reply:
https://github.com/brave/brave-core/pull/15617/commits/e2a76050ea88794a45f742328f6971a8c2694af8